quad_pyblish_module/plugins/tvpaint/publish/extract_psd.py

```diff
@@ -55,7 +55,7 @@ def process(self, instance):
                 files = repre['files']
 
             new_filenames = []
-            for filename in files:
+            for offset, filename in enumerate(files):
                 new_filename = os.path.splitext(filename)[0]
```

Comment:
You should use `Path` from **pathlib**, and for that line this should be `Path(filename).stem`.